src/pages/Login/Login.js

```diff
@@ -1,7 +1,182 @@
-import React from 'react';
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import Modal from 'react-modal';
 
 function Login() {
-  return <div>Login</div>;
+  const [modalOpen, setModalOpen] = useState(false);
+
+  const ModalHandler = () => {
```

Comment:
```suggestion
  const modalHandler = () => {
```
특정 경우 제외하고는 camelCase로 네이밍 지어주세요!